src/main/resources/liquibase/2.6.x.x/critical_range.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+	xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+	  http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+	<property name="now" value="now()" dbms="postgresql" />
+
+	<changeSet author="cliff" id="1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="result_limits" columnName="low_critical_range" />
+			</not>
+		</preConditions>
+		<addColumn schemaName="clinlims" tableName="result_limits">
+			<column name="low_critical_range" type="double precision" defaultValueComputed="'-Infinity'::double precision"/>
+		</addColumn>
+	</changeSet>
+
+    <changeSet author="cliff" id="2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="result_limits" columnName="high_critical_range" />
+			</not>
+		</preConditions>
+		<addColumn schemaName="clinlims" tableName="result_limits">
+			<column name="high_critical_range" type="double precision" defaultValueComputed="'Infinity'::double precision"/>
+		</addColumn>
+	</changeSet>
+
```

Comment:
@caseyi says that we should expect up to 2 critical ranges per test, so we will need up to 4 values to track this. highRangeHigh, highRangeLow, lowRangeHigh, lowRangeLow. If it is easier for you to make the backend handle a variable number of ranges, please feel free to do so, otherwise add the 2 other fields

Reply:
in the example given of 95+ and 0-10 it would be the following:
highRangeHigh=Infinity
highRangeLow=95
lowRangeHigh=10
lowRangeLow=0

Reply:
@CalebSLane kindly review ,, thanks